x/evm/statedb/statedb.go

```diff
@@ -311,7 +311,9 @@ func (s *StateDB) setStateObject(object *stateObject) {
 }
 
 func (s *StateDB) restoreNativeState(ms sdk.MultiStore) {
-	s.cacheCtx = s.cacheCtx.WithMultiStore(ms)
+	manager := sdk.NewEventManager()
+	s.cacheCtx = s.cacheCtx.WithMultiStore(ms).WithEventManager(manager)
```

Comment:
this will clear all the native events, right?

Reply:
yes